qiskit_experiments/analysis/data_processing.py

```diff
@@ -63,10 +67,11 @@ def mean_xy_data(
       :math:`\sigma^2 = 1 / (\sum_{i=1}^N 1 / \sigma_i^2)`
 
     Args
-        xdata: 1D or 2D array of xdata from curve_fit_data or
+        xdata: 1D array of xdata from curve_fit_data or
                multi_curve_fit_data
         ydata: array of ydata returned from curve_fit_data or
                multi_curve_fit_data
+        series: 1D int array that specifies the data series
```

Comment:
I once wrote a comment in #19 about this line. I find it difficult to understand what "1D int array that specifies the data series" means.